City-of-Helsinki/helsinki-paatokset
# PP-320: HDBT syncing vol 2

Fix issues stemming from hdbt-update.

Ticket: https://helsinkisolutionoffice.atlassian.net/jira/software/c/projects/PP/boards/172?modal=detail&selectedIssue=PP-320

To test;
- Run `drush cim -y; drush cr;`
- Go to [Helsinki Kanava settings.](https://helsinki-paatokset.docker.so/fi/admin/config/helsinki-kanava/settings) Toggle debug mode on.
- Make sure you have at least some policymaker-related data on loca (policymakers, meetings, decisions).

Try adding at least these paragraphs to frontpage:
- Meetings calendar
- List of links

Check that the frontpage looks OK.

Check these 2 pages:
- https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto
- https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginhallitus

On kaupunginvaltuusto page, make sure that the announcement look OK.
Make sure that the styles on pages look OK. Check that the subpages work and styles in there look OK.

Check the search pages
- https://helsinki-paatokset.docker.so/fi/paattajat
- https://helsinki-paatokset.docker.so/fi/asia

Check that the styles look OK here as well.
